test-e2e/base-operator-install.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - github.com/ansible/awx-operator/config/default?ref=1.0.0
```

Comment:
At some point, we may want to find a way to automatically use the latest available awx-operator release (https://github.com/ansible/awx-operator/releases).  

I don't think that should block merging this though.